test/operations_test.cpp

```diff
@@ -253,6 +257,11 @@ namespace
     fs::path to;
   };
 
+  void bad_creation_time()
+  {
+    fs::creation_time(".");
```

Comment:
Why is this guaranteed to throw? Is this a portable assumption? I could imagine, it should return the creation time of the current directory.

If you want to simulate error, try passing a path to a non-existing file, for example.